dokuwiki/lib/plugins/doodle/syntax.php

```diff
@@ -599,6 +604,11 @@ function getDoodleFileName() {
           return 'doodle.doodle';
         }
         $dID       = hsc(trim($this->params['title']));
+        if($this->params['anonymous']) {
+            // different filename for anonymous polls, so that it won't be possible
+            // to reveal anonymous poll data by making it not anonymous
+            $dID .= ".a" . md5($dID);
```

Comment:
Not too fussed about this, but doesn't this already reveal the identifier by just running a simple hash over the list of existing user identifiers?